app/Models/Genre.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Genre extends Model
+{
+    use HasFactory;
+
+    public function GenreBooks(){
```

Comment:
Fix the naming and put the `{` on the next line.

Reply:
Fix in other places too.

Reply:
done